core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -101,6 +101,41 @@ angular.module('oppia').component('contributorDashboardPage', {
         ctrl.userCanReviewTranslationSuggestionsInLanguages = [];
         ctrl.userCanReviewVoiceoverSuggestionsInLanguages = [];
         ctrl.userCanReviewQuestions = false;
+        ctrl.defaultHeaderVisible = true;
+        ctrl.collapsibleHeaderVisible = false;
+        ctrl.scrollTest = false;
+
+        var $win = angular.element($window);
```

Comment:
Is this necessary? Can we use the `pageYOffset` property of `$window` instead?

Reply:
Done